src/i18n/eng.json

```
@@ -46,8 +46,8 @@
   "opening_soon": "Opening soon!",
   "for_distribution": "for distributing",
   "for_receiving": "for receiving",
-  "seeking_volunteers": "Seeking Volunteers",
-  "seeking_money": "Seeking Money",
+  "seeking_volunteers": "Volunteer Here",
```

```suggestion
  "seeking_volunteers": "Volunteers Needed Here",
```
Sorry for taking so long to get back to you @omawhite ! I talked to people in the organization and they preferred "Volunteers Needed Here" to "Volunteer Here."